elements/rh-pagination/rh-pagination.ts

```diff
@@ -113,46 +118,69 @@ export class RhPagination extends LitElement {
     super.update(changed);
   }
 
+  #internalColorPalette?: string | null;
+
+  protected willUpdate(): void {
+    /**
+     * TEMPORARY: this fixes the need to access the parents color-palette in order to get the `lightest`
+     * value. This fix will only update the component when switching between light and dark themes as
+     * thats when the consumer requests an update. Switching between lighter -> light for example will
+     * not trigger the component to update at this time. Related: #1395.
+     */
+    this.#internalColorPalette = this.closest('[color-palette]')?.getAttribute('color-palette');
```

Comment:
@bennypowers we stripped this idea out of switch, it still exists in table.  What are your thoughts here?

Reply:
Ok the issue here is the same one we discussed on a call a ways back

Because we have not yet fully developed our theming process yet, it seems to me that shipping this will actually break theming going forward. 

Imagine that the user implements a bordeaux-and-rosewater colour theme for their page, then adds a paginator on a `lighter` container - they'll get paginator colours that assume stock surface colours, and it won't look as expected on the `lighter` rosewater surface colour.
 
For that reason may I suggest that we limit pagination to being used on `lightest` and `darkest` containers only for the time being, then circle back when we work on theming to address `lighter` `light` `dark` and `darker`?

Thanks to @adamjohnson for talking this out with me earlier today.

Reply:
Talked about this with @marionnegp. She's okay with this. So, I went ahead and removed light/lighter/dark/darker colors (and docs) so that we are just sticking with the normal darkest ("`.dark`")/lightest color palette.

Resolved in [63a2749](https://github.com/RedHat-UX/red-hat-design-system/pull/1587/commits/63a274962a434280c12d996b0e66399952977fc5).

FWIW, the only situation where the background color of the box variant bleeds into the surface background color is "Lighter" in the color context demo. I added a note in the docs suggesting consumers place Paginations on lightest and darkest surfaces to avoid confusion.